README.md

```diff
@@ -214,6 +375,28 @@ The following is an example of an OCI image index sent to the registry.
 }
 ```
 
+The first manifest in the manifest list references the CNAB configuration. An
+example of this follows:
+
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "config": {
+    "mediaType": "application/vnd.oci.cnab.config.v1+json",
+    "digest": "sha256:4bc453b53cb3d914b45f4b250294236adba2c0e09ff6f03793949e7e39fd4cc1",
+    "size": 578
+  },
+  "layers": []
+}
+```
+
+Subsequent manifests in the manifest list are standard OCI images.
+
+This example proposes two OCI specification and registry changes:
+1. It proposes the addition of an `org.opencontainers.artifactType` annotation to be included in the OCI specification.
```

Comment:
This is where we're centering on `manifest.config.mediaType` as the declaration of it's type